frida_tools/tracer.py

```diff
@@ -62,7 +63,11 @@ def _initialize(self, parser, options, args):
             self._tracer = None
             self._targets = None
             self._profile = self._profile_builder.build()
-            self._quiet = options.quiet
+            self._quiet = options.quiet == True
+            self._output = options.output
+            if self._output is not None:
+                with OutputFile(self._output, truncate=True) as output:
```

Comment:
We should probably just store the `OutputFile` instance on `self` so we don't open and close the file all the time.

Reply:
yeah, this outputfile class is pretty messy. i have simplified the whole thing. can you review again?

Reply:
Sweet! Will do.